### ISSUE TITLE
[FR] Keybinding to turn on/off mapper

### ISSUE THREAD

**Issue by Preetpal123** [srevinsaju/guiscrcpy#236]
Everytime I press escape key it abort mapping.
How can it be controlled or to set Keybinding to turn it on and off.

I've also tried to set mod key but couldn't find a way. It only assigns characters and numerics as keybindings which make it limited and they always keep working even if we are out of mirroring window.

Plz guide how to bind ctrl/alt combinations and control mapping behaviour outside mirroring to stop uncontrolled mirror screen movements.

**Comment by srevinsaju** [comment]
As of now.. there is no way to use modifier keys like ctrl, alt..

**Comment by Preetpal123** [comment]
> As of now.. there is no way to use modifier keys like ctrl, alt..

I've disabled escape to abort mapping from /use/lib/python3.9/site-packages/guiscrcpy/lib/mapper/mapper.py

What about turning on off mapping from Keybinding?

**Comment by srevinsaju** [comment]
> What about turning on off mapping from Keybinding?

I did not understand, what is it that you want to achieve?

**Comment by Preetpal123** [comment]
> > What about turning on off mapping from Keybinding?
> 
> I did not understand, what is it that you want to achieve?

Actually, even when I'm outside mirror screen these mapping keep working always. For e.g I've set j and l to move 10 seconds ahead or backward but even when I'm working on other project and press j or l it applies to mirroring screen.

**Comment by srevinsaju** [comment]
Agreed, there is no way to prevent that as of now. The "mapping screen" (originally known as `scrcpy`)  is not something what I created. It's source code is over [here](https://github.com/Genymobile/scrcpy). You might want to ask them about adding keyboard mapping support to scrcpy itself..
